services/cartService.js

```diff
@@ -0,0 +1,25 @@
+const cartDao = require("../models/cartDao");
+
+const insertData = async (
+  userId,
+  plantId,
+  plantQuantity,
+  potId,
+  potQuantity,
+  nutrientId,
+  nutrientQuantity
+) => {
+  return cartDao.insertData(
```

Comment:
```suggestion
  return cartDao.createCart(
```
method 이름만 보고도 알 수 있게 수정해주세요!